README.md

```diff
@@ -108,6 +107,11 @@ The configuration of benchmarks allows you to select the frameworks to run, sele
 |**[GradientBoostingClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html)**|gbt|:x:|:x:|:x:|:white_check_mark:|
 |**[GradientBoostingRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html)**|gbt|:x:|:x:|:x:|:white_check_mark:|
 
+## Intel(R) Extension for Scikit-learn* support
+[Intel(R) Extension for Scikit-learn](https://github.com/intel/scikit-learn-intelex) providing drop-in patching speeds up scikit-learn on CPU and GPU. 
```

Comment:
"providing drop-in patching": there is no final decision yet about how XPUs will be supported. Can we use more neutral wordings like "provides optimized functionality of scikit-learn on Intel(R) CPU and GPU" at the moment?